src/common/selectors.ts

```diff
@@ -0,0 +1,246 @@
+import type { Page } from "@playwright/test";
+
+export const selectors = {
+    dashboard: {
+        parent: "dashboard",
+        elements: {
+            target: "div[role=\"main\"] >> text=Clear and preload cache"
+        }
+    },
+    cache: {
+        parent: "cache",
+        elements: {
+            mobileDeviceCache: {
+                checkbox: "#cache_mobile",
+                target: "label[for=cache_mobile]"
+            },
+            mobileDeviceSeparateCache: {
+                checkbox: "#do_caching_mobile_files",
+                target: "label[for=do_caching_mobile_files]"
+            },
+            cacheLoggedUser: {
+                checkbox: "#cache_logged_user",
+                target: "label[for=cache_logged_user]"
+            }
+        }
+    },
+    fileOptimization: {
+        parent: "file_optimization",
+        elements: {
+            minifyCss: {
+                checkbox: "#minify_css",
```

Comment:
I am wondering what you gonna execute if a user create that invalid state: 
```json 
{
   checkbox: "#do_caching_mobile_files",
   textbox: "#cache_reject_cookies"
}
```

It wouldn't be easier to add a field `type` that would define which logic we gonna execute?